h/templates/client/annotation.html

```diff
@@ -96,8 +96,8 @@
   <div class="annotation-body tags tags-read-only"
        ng-if="vm.form.tags.length && !vm.editing()">
     <ul class="tag-list">
-      <li class="tag-item" ng-repeat="tag in vm.form.tags">
-        <a href="/stream?q=tag:'{{tag.text|urlencode}}'" target="_blank">{{tag.text}}</a>
+      <li class="tag-item" ng-repeat="tag in ::vm.form.tags">
+        <a href="/stream?q=tag:'{{::tag.text|urlencode}}'" target="_blank">{{::tag.text}}</a>
```

Comment:
This breaks live-update of tags, when the annotation is edited by another client